### DISCUSSION TITLE
Code progress

### DISCUSSION THREAD

**Discussion by longjulia** [Jo-Lo/Tafe22S203#2] [General]
How is everyone doing with their c# and xaml codes for their calculators? If you need ideas or get stuck, comment in this discussion!

**Comment by Jo-Lo** [comment]
I got my C# and xaml code working. Adding a few adjustments to the layout to make it more appealing.

**Comment by ghost** [comment]
Same for me here, it is working fine so far, just need to modify a part of my C#

**Comment by longjulia** [comment]
Right on everyone! 

I have a question: What code is written for the "exit" button? Can someone share it? Cause that part I don't know. ** Just clarifying that I understood the exit button as something that went back to the main menu.

**Reply by ghost** [reply]
Hey, I will copy and paste what I put for the exit button: 
this.Frame.Navigate(typeof(GIT_Assignment_Main_Menu.MainMenu));
I hope this helps!

**Comment by ghost** [comment]
Hey all, do you guys want a specific color scheme for the calculators or do we all give it our "personal touch" ?

**Reply by longjulia** [reply]
I think we should use our own colour schemes. Let's not limit our *creativity*. I'm for giving it our personal flair

**Reply by ghost** [reply]
Ok cool  think same goes for the fonts etc. then?

**Reply by longjulia** [reply]
same goes! go crazy

**Reply by Jo-Lo** [reply]
I agree, we should use our own theme of colours because more variety brings more options to experiment with.
also adobecolor.com is a great resource for colour theme ideas

**Reply by Jo-Lo** [reply]
with our fonts I think we use our own choice for that personal touch.

**Comment by ghost** [comment]
One more question from me, for the buttons besides the "calculate" and 'exit' button is there additional one you would like me to add? i also added a "clear" button for the user to clear their input.

**Reply by longjulia** [reply]
Very good thinking! That means we should have 3 buttons: Calculate, Clear, Exit. We should all try to implement your idea

**Reply by Jo-Lo** [reply]
oh yes, cool idea I'll try to implement the clear button.
